src/components/Forms/ReviewForm/ReviewForm.js

```diff
@@ -88,6 +89,11 @@ const ReviewForm = props => {
   );
 };
 
+ReviewForm.propTypes = {
+  className: PropTypes.string,
+  setOptions: PropTypes.func
```

Comment:
propTypes are wrong. To check what kind of proptypes it is receiving do CTRL + SHIFT + F search for the component name and see what props it is receiving